test/e2e/tests/custom-token-send-transfer.spec.js

```diff
@@ -0,0 +1,272 @@
+          WINDOW_TITLES.ExtensionInFullScreenView,
+        );
+        await driver.clickElement({
+          css: '[data-testid="multichain-token-list-item-value"]',
+          text: '10 TST',
+        });
+        await driver.clickElement('[data-testid="eth-overview-send"]');
+        await driver.fill(
+          'input[placeholder="Enter public address (0x) or ENS name"]',
+          recipientAddress,
+        );
+        await driver.fill('.unit-input__input', '1');
+        await driver.clickElement('[data-testid="page-container-footer-next"]');
+
+        // check transaction details
+        await driver.waitForSelector({
+          text: '1 TST',
+          tag: 'h1',
+        });
+        await driver.waitForSelector({
+          text: 'Transfer',
+          css: '.confirm-page-container-summary__action__name',
+        });
+        const estimatedGasFee = await driver.findElements(
+          '.currency-display-component__text',
+        );
+        assert.notEqual(
+          await estimatedGasFee[0].getText(),
+          '0',
+          'Estimated gas fee should not be 0',
+        );
+
+        // check function name and hex data details in hex tab
+        await driver.clickElement({
+          text: 'Hex',
+          tag: 'button',
+        });
+        await driver.waitForSelector({
+          text: 'Transfer',
+          tag: 'span',
+        });
+        await driver.waitForSelector({
+          tag: 'p',
+          text: '0xa9059cbb0000000000000000000000002f318c334780961fb129d2a6c30d0763d9a5c97',
+        });
+
+        // edit gas fee
+        await driver.clickElement({ text: 'Details', tag: 'button' });
+        await driver.clickElement({ text: 'Edit', tag: 'button' });
+        await editGasfeeCustomToken(driver);
+        await driver.clickElement({ text: 'Confirm', tag: 'button' });
+
+        // check that transaction has completed correctly and is displayed in the activity list
+        await driver.waitForSelector({
+          css: '[data-testid="activity-list-item-action"]',
+          text: 'Send TST',
+        });
+        await driver.waitForSelector(
+          {
+            css: '.transaction-list__completed-transactions [data-testid="transaction-list-item-primary-currency"]',
+            text: '-1 TST',
+          },
+          { timeout: 10000 },
+        );
+      },
+    );
+  });
+
+  it('transfer custom tokens from dapp customizing gas values', async function () {
+    await withFixtures(
+      {
+        dapp: true,
+        fixtures: new FixtureBuilder()
+          .withPermissionControllerConnectedToTestDapp()
+          .build(),
+        ganacheOptions,
+        smartContract,
+        title: this.test.fullTitle(),
+      },
+      async ({ driver, contractRegistry }) => {
+        const contractAddress = await contractRegistry.getContractAddress(
+          smartContract,
+        );
+        await unlockWallet(driver);
+
+        // on testdapp, add created tokens in wallet
```

Comment:
same as above. Adding  the token in the state with fixtures would allow us to skip this step

Reply:
Done, thanks !